lib/src/io_file.dart

```diff
@@ -230,6 +261,57 @@ class _DataFile implements IOFile {
   }
 }
 
+/// `IOFile` implementation with the given `bytes`.
+class _StreamFile implements IOFile {
+  _StreamFile(
+    this._openReadStream, 
+    this._length, {
+    required this.contentType,
+    required this.lastModifiedDate,
+    required this.name,
+    required this.path,
```

Comment:
Indeed, as the path will always be empty, it could be cleaner if we remove this from the constructor, and instead do
```dart
final String path = '';
```
Or perhaps a getter that throws? It depends on how it's being used.

Reply:
[2c9ed88](https://github.com/ar-io/ardrive_io/pull/8/commits/2c9ed88ca173558591435c216e3b4d483a5b9889)